types/react-payment-inputs/index.d.ts

```diff
@@ -48,7 +48,7 @@ interface CardImages {
 
 type InputValidationErrorCallback = (error: string, erroredInputs: ErroredInputs) => void;
```

Comment:
This type can probably be exported as well if everything else is:

```suggestion
export type InputValidationErrorCallback = (error: string, erroredInputs: ErroredInputs) => void;
```

If this type is exported, the `export {}` line [here](https://github.com/tyirenkyi/DefinitelyTyped/blob/8644c2e78c64ad5e2e4800dc9bf9a570d88fdcc5/types/react-payment-inputs/index.d.ts#L316) can be removed.